runner/cf-driver-go/cmd/root.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/GSA-TTS/gitlab-runner-cloudgov/runner/cf-driver/cmd/drive"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(drive.DriveCmd)
+}
+
+var rootCmd = &cobra.Command{
+	Use:   "cfd",
```

Comment:
[question] - the docs mention `cfd` but when I run `make build` I get a `cf-driver` executable. Should those be consistent? If something needs to change I do like the shorter version more.

Reply:
Good catch, I haven't been using the actual executable much & didn't think of it.